docs/tre-templates/pipeline-templates/overview.md

```diff
@@ -0,0 +1,56 @@
+# Pipeline Templates
```

Comment:
Nice work on updating docs right away 👍 
I think maybe make it super clear that it's a work in progress for now, so that people don't try to update things and expect it to work?